core/internal/testutils/testutils.go

```diff
@@ -31,8 +31,11 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
 	// NOTE: To avoid circular dependencies, this package MUST NOT import
 	// anything from "github.com/smartcontractkit/chainlink/v2/core"
+
+	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/p2pkey"
```

Comment:
Maybe we should have a `package keystest` to keep this package simpler :thinking:

Reply:
Done